YvesIraguha/readit_react_front_end
# #165547961 Set up Testing Environment

#### What does this PR do?
- Set up a testing environment
#### Description of Task to be completed?
- Create a dummy test
- Transform all pages from HTML to react 
- Create reducers and action creators. 
- Create shared and basic components
#### How should this be manually tested?
- Clone the repo and check out the branch `ch-setup-testing-environment-165547961`
- Install all dependencies using `yarn`
- Run`yarn test`
- Start your server by `yarn start`, you should home pages
- Navigate to different pages. 

# 165547961
